src/extensions/filter-control/utils.js

```diff
@@ -225,7 +225,18 @@ export function isFilterDataNotGiven ({ filterData }) {
 export function hasSelectControlElement (selectControl) {
   return selectControl && selectControl.length > 0
 }
-
+export function getObjectValueByString (obj, memberpath) {
```

Comment:
I think we could move this to Utils file.. Not this one, the main one

Reply:
Thanks for the pointer, I checked src/Utils/index.js and actually found a function that does what we need.

I have no overview over the codebase so I didn't know it existed.

I have removed my function and used the one already in global utils.

The change is now a one-liner 👍